File: src/main/java/br/com/iem/model/Entry.java
package br.com.iem.model;

import java.time.LocalDate;

import java.math.BigDecimal;

import lombok.Data;

@Data
public class Entry {
	private Long id;
	private String name;
	private String description;
	private LocalDate date;
	private BigDecimal value;
}

File: src/domain/model/Entry.js
export default class {
  id;
  name; 
  description;
  value;
  date;
  account;

  constructor({ id, name, description, value, date, account }) {
    this.id = id;
    this.name = name;
    this.description = description;
    this.value = value;
    this.date = date;
    this.account = account;
  }
}